app/controllers/reports_controller.rb

```diff
@@ -152,6 +152,9 @@ def set_report
   end
 
   def set_user_hash
+    if current_user&.uid.blank?
+      raise CanCan::AccessDenied.new("Not authorized")
```

Comment:
This should already be covered by:
https://github.com/datacite/sashimi/blob/master/app/controllers/reports_controller.rb#L17
That should raise the cancan exception, typically you dont raise it yourself.
But perhaps the order is wrong for when the JWT is decoded.

Reply:
@richardhallett seems that authorize_resource seems to pass even when no token is provided. I've rather bumped the gem version this seems to give me the NoMethodError instead of a 500 error.